src/biokbase/narrative/clients.py

```diff
@@ -19,20 +19,20 @@ def reset():
 def __init_client(client_name, token=None):
     if client_name == "workspace":
         c = Workspace(URLS.workspace, token=token)
-    elif client_name == "narrative_method_store":
-        c = NarrativeMethodStore(URLS.narrative_method_store, token=token)
-    elif client_name == "user_and_job_state":
-        c = UserAndJobState(URLS.user_and_job_state, token=token)
-    elif client_name == "catalog":
-        c = Catalog(URLS.catalog, token=token)
-    elif client_name == "service" or client_name == "service_wizard":
-        c = ServiceClient(URLS.service_wizard, use_url_lookup=True, token=token)
     elif (
         client_name == "execution_engine2"
         or client_name == "execution_engine"
         or client_name == "job_service"
     ):
         c = execution_engine2(URLS.execution_engine2, token=token)
+    elif client_name == "narrative_method_store":
```

Comment:
Reordered the clients in rough order of how often they are used (micro-optimisations FTW!)

Is there a reason to keep these other client names around when the codebase only uses one of the variants? e.g. `execution_engine2` for EE2 or `service` for the service wizard

Reply:
Yes! They're super useful as a narrative user. I use them all the time to just let the narrative be my tinkering / coding environment.

These were really put in as an early step toward exposing code cells for general usage, but that project kinda froze a while ago.

Reply:
Oh, right, the variants. I don't think that matters as much. The internal codebase should be kept consistent, I'm okay with ditching them.